setup.py

```diff
@@ -4,9 +4,9 @@
     long_description = fh.read()
 
 setup(
-    name="timing_analysis",
+    name="pint_pal",
     version="1.0.0",
-    description="NANOGrav Timimg Analysis",
+    description="NANOGrav Pulsar Timimg Analysis Software",
```

Comment:
Maybe something like "Notebook/PINT-based pulsar timing analysis software" instead?

Reply:
FWIW I agree with adding something about the Notebooks to the description because that's one of the really big advantages introduced here, right?